app/javascript/react/screens/App/Overview/components/Migrations/MigrationsInProgressCard.js

```diff
@@ -208,24 +208,28 @@ const MigrationsInProgressCard = ({
 
   return (
     <InProgressWithDetailCard plan={plan} failedOverlay={failedOverlay} handleClick={handleClick}>
-      <UtilizationBar
-        now={totalMigratedDiskSpace}
-        max={totalDiskSpace}
-        description={__('Datastores')}
-        label={diskSpaceBarLabel}
-        descriptionPlacementTop
-        availableTooltipFunction={availableDiskSpaceTooltip}
-        usedTooltipFunction={usedDiskSpaceTooltip}
-      />
-      <UtilizationBar
-        now={completedVMs}
-        max={totalVMs}
-        description={__('VMs')}
-        label={vmBarLabel}
-        descriptionPlacementTop
-        availableTooltipFunction={availableVmTooltip}
-        usedTooltipFunction={usedVmTooltip}
-      />
+      <div id="datastore-progress-bar">
+        <UtilizationBar
+          now={totalMigratedDiskSpace}
+          max={totalDiskSpace}
+          description={__('Datastores')}
+          label={diskSpaceBarLabel}
+          descriptionPlacementTop
+          availableTooltipFunction={availableDiskSpaceTooltip}
+          usedTooltipFunction={usedDiskSpaceTooltip}
+        />
+      </div>
+      <div id="vm-progress-bar">
```

Comment:
It would be a good idea to indicate here that the `div id` is for a particular plan id, in case there are many in-progress migrations running at the same time

Reply:
Good point, and we don't want duplicate IDs in the document! Thanks.